src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+---
+title: Resources
+description: Useful resources and community projects
+section: Community
+layout: layout-pane.njk
+order: 1
+status: Experimental
+---
+
+A curated list of community developed resources that can help you use the GOV.UK Design System in your chosen design tool or development environment. 
+
+The Design System team is not responsible for these resources and is unable to provide support for them. If you have support or feature requests please contact the owner.
+
+
+## Create flow diagrams
+
+[GOV.UK Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams)⁠ -  
```

Comment:
The lists don’t seem to have an order. Could you reshuffle them so they’re alphabetical?

Reply:
Also we’d probably need to avoid the manual line breaks after each tool name (it feels a bit like creating ‘subheadings’ but in a non-semantic way). The hyphens alone should work fine I think.